apps/nowcasting-app/styles/globals.css

```diff
@@ -1,3 +1,6 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+.recharts-surface{
+  overflow: visible;
+}
```

Comment:
new line

Reply:
can be replaced with tailwind class "overflow-visible"

Reply:
this is applied to an element created by rechart we don't have access to, were you able to add the "overflow-visible" class? , it would be great if we can keep all styling using tailwind classes